src/components/button/index.md.njk

```diff
@@ -40,6 +31,12 @@ Only use disabled buttons if research shows it makes the user interface easier t
 
 {{ example({group: 'components', item: 'button', example: 'disabled', html: true, nunjucks: true, open: true}) }}
 
+### Start buttons
+
+Use a start button as the main call to action on your service’s start page.
```

Comment:
Is this guidance useful, do we expect services to use the start page?

Or should we be clear this is only used for GOV.UK Publishing?